src/middlewared/middlewared/service.py

```diff
@@ -516,3 +516,39 @@ def sleep_fn():
             import pydevd
             pydevd.stoptrace()
             pydevd.settrace(host=options['host'])
+
+    @accepts(Str("method"), List("params"))
+    @job(lock=lambda args: f"bulk:{args[-1]}")
```

Comment:
`args[-1]` will lock based on `params`. We probably want `args[0]` which is `method`.

Reply:
Why do we want to lock at all? Won't `method` do locking itself if it's needed?